### ISSUE TITLE
Please re-enable Gitpod prebuilds to skip 20 minutes of waiting

### ISSUE THREAD

**Issue by jankeromnes** [forem/forem#9627]
**Is your feature request related to a problem? Please describe.**

When I open Forem in Gitpod, the dev setup is fully automated (which is fantastic), but the setup is not executed ahead-of-time, so I have to wait a long time before my workspace is fully compiled and ready.

**Describe the solution you'd like**

Please configure https://github.com/apps/gitpod-io for https://github.com/forem/forem to enable prebuilt workspaces (in just a few clicks).

This will make Gitpod pre-build Forem workspaces continuously on every push, and will **save everyone up to 20 minutes** every time they start a new workspace:

<img width="601" alt="Screenshot 2020-08-03 at 09 57 21" src="https://user-images.githubusercontent.com/599268/89159962-8dc87700-d570-11ea-8950-d7c84f63f4c5.png">

(Screenshot from @corneliusludmann's fork https://github.com/corneliusludmann/forem where prebuilds are enabled.)

**Describe alternatives you've considered**

The alternative is for every Forem developer, maintainer and contributor to wait up to 20 minutes every time they start a new workspace, which would be sad.

**Additional context**

All credit goes to @corneliusludmann who already optimized the `.gitpod.yml` tasks for Gitpod prebuilds in https://github.com/forem/forem/pull/9560.

Also, Gitpod prebuilds were enabled for the previous repo location in #2380.

**Comment by joshpuetz** [comment]
Tagging @forem/repo-admin to make this change

**Comment by mstruve** [comment]
The Gitpod app was configured and hooked up to the Forem repo which according to the docs https://www.gitpod.io/docs/prebuilds/ makes it seem like it should "just work" and prebuild. I am not seeing other options when I configure the app beyond Read and Write access for the repo. I think the transfer might have borked something so I uninstalled and reinstalled the Gitpod, waiting on repo owner approval to install the Github Apps so we can prebuild. Will keep everyone posted!

**Comment by jankeromnes** [comment]
Many thanks @mstruve for looking into this! I think uninstalling & reinstalling did the trick, as we're seeing Forem prebuilds triggered on our side:

```
+----------------------------+-------------------------------------+-----------+-------------+------------------------------------------+
| creationTime               | cloneURL                            | state     | error       | commit                                   |
+----------------------------+-------------------------------------+-----------+-------------+------------------------------------------+
| 2020-08-07 15:05:27.535053 | https://github.com/forem/forem.git  | building  |             | 84d782c5ba1555d78eeb8c29f72e8f5fb847629b |
| 2020-08-07 15:05:27.094305 | https://github.com/forem/forem.git  | building  |             | 84d782c5ba1555d78eeb8c29f72e8f5fb847629b |
| 2020-08-07 15:04:40.142034 | https://github.com/forem/forem.git  | building  |             | 3c5b01d490e9603baacbfe8018f04580466c62c9 |
| 2020-08-07 15:04:14.007523 | https://github.com/forem/forem.git  | available |             | 263df04d620381d2fee9579f53d99d8a1a39767c |
| 2020-08-07 15:01:45.161399 | https://github.com/forem/forem.git  | available |             | 1c566e0ec44a5ca15704fd3809795d5f66b02c7f |
| 2020-08-07 15:01:04.240661 | https://github.com/forem/forem.git  | available |             | b8f92e6fc5d91932dcdbbfa7083cd6ef905b55b6 |
| 2020-08-07 14:42:41.866112 | https://github.com/forem/forem.git  | available |             | a5ee17ade9a0d7dbd4df2ce82faec0226d599fc4 |
| 2020-08-07 14:27:05.308161 | https://github.com/forem/forem.git  | available |             | 600fd40cca139162bfce487cb16900fe8550b231 |
| 2020-08-07 14:22:03.651661 | https://github.com/forem/forem.git  | available |             | da53933fd4fb39408214c5d3ae9dc8d82b948bc5 |
| 2020-08-07 14:21:19.823927 | https://github.com/forem/forem.git  | available |             | aa268d5a8d462d51f2ffbd4c0286eb4f77914b1e |
+----------------------------+-------------------------------------+-----------+-------------+------------------------------------------+
```

🎉

And indeed trying to open Forem in Gitpod right now shows the latest prebuild is in progress:

![Screenshot 2020-08-07 at 17 14 16](https://user-images.githubusercontent.com/599268/89660325-7cb19a00-d8d1-11ea-9e46-2c082e6f1b8a.png)

FYI, it's always safe to click on `Skip Prebuild` -- this doesn't cancel anything, it just gives you a non-prebuilt Gitpod workspace faster (that can be useful e.g. if you want to quickly access the code but you don't need any dependencies or build results). The prebuild itself will continue undisturbed until successful, allowing subsequent workspaces to instantly get a fully-prebuilt Forem application. 🚀
